configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
+
+There are several existing solutions, that can help you configuring your openHAB instance in a textual way.
+Mainly there are two bigger projects for this. 
+_Eclipse Smart Home - Designer_ as a standalone editor and _openhab-vscode_ as an extension for the Visual Studio Code open source editor.
+Additionally there some syntax highligting solutions existing for other text editors.
+This documentation page can give you some guidance in choosing the right one for you and setting it up.
+
+{::options toc_levels="2"/}
+
+* TOC
+{:toc}
+
+## openhab-vscode
+
+openhab-vscode is an extension for the [Visual Studio Code](https://code.visualstudio.com) editor.
```

Comment:
Could you add a screenshot here? @kubawolanin already contributed a few, I'd prefer a simple png over a gif here for uniformity (I'm ready to be convinced otherwise)